.storybook/components/Toc.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
```

Comment:
This component was copied over from https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/DocsComponents/Toc.stories.tsx and will be removed once original component will be published as a part of a new package.